client/app/components/Dropdown/DropdownFactory.jsx

```diff
@@ -23,7 +23,7 @@ export const DropdownFactory = (variationClassName: string) => {
     value: propValue,
   }: Props) => (
     <select
-      className={`${css.dropdown} ${variationClassName}`}
+      className={window.navigator.appVersion && `${css.dropdown} ${variationClassName}`}
```

Comment:
Could you clarify that this change does?

Reply:
It also looks like you haven't rebased from master in a while, this file actually no longer exists.

Reply:
I was attempting to get the OS on which the browser was running and whichever it was also use the css. However, it was guesswork since I never used OS compatibility for javascript before.